spec/rails_helper.rb

```diff
@@ -70,4 +70,10 @@
   config.filter_rails_from_backtrace!
   # arbitrary gems may also be filtered via:
   # config.filter_gems_from_backtrace("gem name")
+  Shoulda::Matchers.configure do |config|
+    config.integrate do |with|
+      with.test_framework :rspec
+      with.library :rails
+    end
+  end
 end
```

Comment:
In order to use Devise in your specs, you'd need to add:

```
  # Use Devise test helpers in controller specs
  config.include Devise::Test::ControllerHelpers, type: :controller
  # Use Devise test helpers in view specs
  config.include Devise::Test::ControllerHelpers, type: :view
  # Use Devise test helpers in feature specs
  config.include Devise::Test::IntegrationHelpers, type: :feature
```

You'll need this in order to sign_in and sign_out and have access to current_user, etc.